src/plib/views/scripts/index/index.phtml

```diff
@@ -3,9 +3,13 @@
 
 echo '<p>' . \pm_Locale::lmsg('pageDescription') . '</p>';
 
-echo '<div id="piechart" style="width: 800px; height: 400px;"></div>';
+if ($this->isEmptyDir) {
```

Comment:
let's write another message if "runningTask".
now you can hide the popup with current task and you will see "directory is empty" but in fact, it is not.

Reply:
OK